CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+## Release 0.6.0
+
+New features:
+- Preliminary work to add capabilities to nodes, so that the UI will
+  be able to send actions down to the probe.
+  [#315](https://github.com/weaveworks/scope/issues/315)
+- Added support for pcap based packet sniffing, to provide bandwidth
+  usage information.
+  [#317](https://github.com/weaveworks/scope/pull/317)
+- Probes now push data to the app, instead of the app requesting it.
+  [#342](https://github.com/weaveworks/scope/pull/342)
+- Allow probe and app to be started independently.
+  [#345](https://github.com/weaveworks/scope/pull/345)
+- Close details pane when changing topology view.
+  [#297](https://github.com/weaveworks/scope/issues/297)
+- Support --probe.foo=bar style flags.
+  [#347](https://github.com/weaveworks/scope/pull/347)
+- Added X-Scope-Probe-ID header to identify probes.
+  [#351](https://github.com/weaveworks/scope/pull/351)
+
+Bug fixes:
+- Updated to work with newer versions of weave.
```

Comment:
"Update scope script to work with master version of Weave, where DNS has been embedded in the router."